### ISSUE TITLE
Webpack breaking change: The extension in the request is mandatory

### ISSUE THREAD

**Issue by Floffah** [babel/babel#12169]
## Bug Report

- [ ] I would like to work on a fix!
i would but laziness you know

**Current behavior**
In the newly released Webpack 5 (released 3 days ago as of writing this), for a lot of files that are required by webpack to work, you get the following error example:
```
ERROR in ../node_modules/@babel/runtime/helpers/esm/toConsumableArray.js 4:0-52
Module not found: Error: Can't resolve './nonIterableSpread' in 'D:\projects\unfinished\app\a-guide\node_modules\@babel\runtime\helpers\esm'
Did you mean 'nonIterableSpread.js'?
BREAKING CHANGE: The request './nonIterableSpread' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ../node_modules/rc-table/es/Table.js 3:0-78 212:24-42 214:34-52
 @ ../node_modules/antd/es/table/Table.js 9:0-51 405:19-33
 @ ../node_modules/antd/es/table/index.js 1:0-28 2:15-20
 @ ../node_modules/antd/es/index.js 56:0-43 56:0-43
 @ dll library
```

- [REPL](babeljs.io/repl), [Codesandbox](https://codesandbox.io/s/babel-repl-custom-plugin-7s08o?file=/src/index.js), or GitHub Repo helps!

**Input Code**

No code is written, this occurs even in the most basic-est of webpack configurations (however mostly occurs in dlls)

**Expected behavior**
The files should be compiled into webpack successfully 

**Babel Configuration (babel.config.js, .babelrc, package.json#babel, cli command, .eslintrc)**
- Filename: `babel.config.js`

there is none

**Environment**

```

```
- Babel version(s): [e.g. v6.0.0, v7.0.0-beta.34] webpack's one
- Node/npm version: [e.g. Node 8/npm 5] node 12.18.4/6.14.6 & 14.13.0/6.14.8
- OS: [e.g. OSX 10.13.4, Windows 10] windows 10
- Monorepo: [e.g. yes/no/Lerna] - dont know what that is so presumed not
- How you are using Babel: [e.g. `cli`, `register`, `loader`] through webpack

**Possible Solution**

**just add file extensions in your imports**

**Additional context**
webpack broke a lot of files lmao

**Comment by JLHwung** [comment]
Duplicate of #12158

**Comment by JLHwung** [comment]
Fixed in `@babel/runtime@7.12.0`.

**Comment by Floffah** [comment]
@JLHwung thank you
